index.html

```diff
@@ -3983,8 +3986,9 @@ <h3>Usage Patterns</h3>
       </p>
 
       <p>
-Privacy risks of <a>credential</a> usage occur when unintended or unexpected
-correlation arises from the presentation of <a>verifiable credentials</a>.
+Privacy risks of <a>verifiable credential</a> usage occur when unintended or
```

Comment:
should probably be credential in both cases.

Reply:
Fixed.